app/models/routable_artefact.rb

```diff
@@ -0,0 +1,87 @@
+require 'plek'
+require 'gds_api/router'
+
+class RoutableArtefact
```

Comment:
This class currently does a lot more than it needs to. But I guess it's a straight lift from Panopticon and will be used for the creation of content as well, so probably fine!